src/coinjoin/context.cpp

```diff
@@ -5,6 +5,7 @@
 #include <coinjoin/context.h>
 
 #include <net.h>
+#include <net_processing.h>
```

Comment:
why this header is needed?

Forward declaration of `PeerManager` should be enough

Reply:
Good catch, removed it and a few other extraneous headers! Resolved in latest push!